docs-chef-io/content/inspec/plugin_kitchen_inspec.md

```diff
@@ -40,6 +40,12 @@ inspec compliance login https://compliance.test --user admin --insecure --token
 
 where `--insecure` is required when using self-signed certificates.
 
+`inspec automate` is an alias for `inspec compliance` and works the same way:
```

Comment:
List automate before compliance, and present compliance as being a backwards-compatible alias.